changelog.txt

```diff
@@ -39,6 +40,7 @@ Fixed
 Deprecated
 ++++++++++
 
+ - get_id in ``Project`` and ``Job`` class (#250)
```

Comment:
```suggestion
 - get_id in ``Project`` and ``Job`` classes has been replaced in favor of the id property (#250).
```

Reply:
Since the entries have a period at the end, they should probably be complete sentences as well. I am aware that this is not true for all previous entries.